src/libstd/Cargo.toml

```diff
@@ -18,7 +18,7 @@ panic_unwind = { path = "../libpanic_unwind", optional = true }
 panic_abort = { path = "../libpanic_abort" }
 core = { path = "../libcore" }
 libc = { path = "../rustc/libc_shim" }
-compiler_builtins = { path = "../rustc/compiler_builtins_shim" }
+compiler_builtins = { path = "../rustc/compiler_builtins_shim", features = ["mem"] }
```

Comment:
Is there a way to do this without explicitly setting the feature for all users?

Reply:
Wow, that's quite the hack to get around https://github.com/rust-lang/cargo/issues/1197